plugins/woocommerce/patterns/coming-soon-entire-site.php

```diff
@@ -8,6 +8,14 @@
  * @package WooCommerce\Blocks
  */
 
+$current_theme     = wp_get_theme()->get_stylesheet();
+$inter_font_family = 'inter';
+$cardo_font_family = 'cardo';
+
+if ( 'twentytwentyfour' === $current_theme ) {
+	$inter_font_family = 'body';
+	$cardo_font_family = 'heading';
+}
```

Comment:
This is required because TT24 uses body and heading as the font's name. I think it's risky to use the same name because I feel it's too common and might conflict with other themes.

This has one downside: If someone customize a page in a different theme, then switches to TT24, or vice versa, things may or may not work as expected (likely impact is font dropdown will be set to Default instead).